src/sources/kafka.rs

```diff
@@ -179,6 +183,22 @@ fn kafka_source(
                                 log.insert(offset_key, Value::from(msg.offset()));
                             }
 
+                            if let Some(headers_key) = &headers_key {
+                                let mut headers_map = BTreeMap::new();
+                                if let Some(headers) = msg.headers() {
+                                    for i in 0..headers.count() {
+                                        let header = headers.get(i).unwrap();
```

Comment:
Could we make this `.expect("header does not exist")`? I see it only returns `None` if the index is out-of-bounds. This is a bit of an odd API that rdkafka is exposing. I would have expected to see it define `Iter`.